This adds /usr/lib/amd64 to the 'Libs' section in the 64-bit gpg-error.pc
and thus fixes the gpg-error-config-test.sh test on amd64.

--- libgpg-error-1.46/configure.ac.orig	2022-11-01 08:28:03.817023292 +0000
+++ libgpg-error-1.46/configure.ac	2022-11-01 08:31:00.432594110 +0000
@@ -547,7 +547,7 @@
     ;;
 esac
 case "$libdir" in
-  /usr/lib|/usr/lib64|/lib|/lib64) ;;
+  /usr/lib|/usr/lib64|/lib|/lib64|/usr/lib/amd64) ;;
   '${exec_prefix}/lib'|'${exec_prefix}/lib64')
     if test "$exec_prefix" = "NONE"; then
       if test "$prefix" != / -a "$prefix" != /usr; then
